assets/src/edit-story/components/rangeInput/index.js

```diff
@@ -46,6 +46,7 @@ export default styled.input.attrs(({ min, max, step, value, onChange }) => ({
   height: 4px;
   appearance: none;
   flex: 1;
+  flex-grow: ${({ flexGrow }) => flexGrow || 0};
```

Comment:
Rather than this, you could also just extend it in `gridview/index.js` as:

```js
const GrowingRangeInput = styled(RangeInput)`
  flex-grow: 1;
`;
```

I think that's a better practice than generalizing components.

Reply:
Okay. I thought it better to have more generic component. 👍